playgrounds/skeleton-svelte/src/routes/components/tabs/+page.svelte

```diff
@@ -139,4 +140,40 @@
 			</Tabs>
 		</div>
 	</section>
+
+	<section class="space-y-4">
+		<h2 class="h2">Anchor-Based Tabs Test</h2>
+		<Tabs value={group} onValueChange={(e) => (group = e.value)}>
+			<Tabs.Control>
+				<a
```

Comment:
This doesn't appear to be valid code. The Tabs control represents the button/anchor. Why would multiple anchors be placed inside this?